x-pack/plugins/apm/public/components/app/dependency_operation_detail_view/dependency_operation_detail_trace_list.tsx

```diff
@@ -190,6 +190,7 @@ export function DependencyOperationDetailTraceList() {
       ),
       field: 'transactionName',
       truncateText: true,
+      width: '60%',
```

Comment:
nit: if the transaction name is long maybe you can use `TruncateWithTooltip`.  

The table and the column's width feels uneven by increasing the width of this column

Reply:
Agreed that it looks odd, I'll see if there's a better way. What would be the benefit of `TruncateWithTooltip` over `truncateText`?

Reply:
The motivation for `TruncateWithTooltip` was that we were truncating text without way for the user to see the full version. This was (and is) problematic.
The problem with `TruncateWithTooltip` is that it shows a tooltip unconditionally: regardless if truncation is in effect or not. It's a little odd to see a tooltip when the exact same is available untruncated on the page already.

Reply:
Per our discussion yesterday, I'll keep the table as-is and remove it in a follow-up unless there are concerns: https://github.com/elastic/kibana/issues/144081.